comparsion-and-logical-operators/3.grading-system.js

```diff
@@ -0,0 +1,17 @@
+var gradeScore = 61;
+
+if (gradeScore < 40) {
+    grade = "F";
+} else if(gradeScore < 50) {
+    grade = "E";
+} else if(gradeScore < 60) {
+    grade = "D";
+} else if(gradeScore < 70) {
+    grade = "C";
+} else if(gradeScore < 80) {
+    grade = "B";
+} else if(gradeScore < 90) {
```

Comment:
This is wrong: 95% is still an 'A'. either else (not else...if) or by declaring the grade as 'A' to start with nothing is needed at all.